Assets/Character/Scripts/PlayerInput.cs

```diff
@@ -4,44 +4,44 @@ public static class PlayerInput
 {
     public static bool PressedRoll()
     {
-        return Input.GetKeyDown(KeyCode.C);
+        return Input.GetKeyDown(KeyCode.C) || Input.GetButtonDown("Roll");

```

Comment:
I used it because Input.GetButtonDown can put multiple positive buttons, so I have "C" as one and a joystick button aka controller button as the other (I've since removed GetKeyDown(KeyCode.C), this was initially added for the xbox controller specifically)